### PR TITLE
sql: do not show FAMILY if there is only one on SHOW CREATE TABLE

### PR DESCRIPTION
**Pull request by otan**
Release note (sql change): SHOW CREATE TABLE no longer shows the FAMILY
clause if there is only the PRIMARY family clause.

### CODE REVIEW

**Review by rafiss — APPROVED** [review]
lgtm if we get PM signoff

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/76285#-:-MvUL9u49NTZh6JtyKDT:bfle0nj)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan and @rafiss)

___
*[pkg/sql/show_create_clauses.go, line 402 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/76285#-MvUL0r33DgLeNlZtniX:-MvUL0r33DgLeNlZtniY:b-u34vom) ([raw file](https://github.com/cockroachdb/cockroach/blob/65deaed84a676a1e0c8c265f519d746971cd4dea/pkg/sql/show_create_clauses.go#L402)):*
> ```Go
> 	// it is named "primary".
> 	families := desc.GetFamilies()
> 	if len(families) == 1 && families[0].Name == "primary" {
> ```

i think there's a const we can use instead of hard-coding the string

**Comment by adityamaru** [comment]
Bulk changes LGTM!

**Comment by otan** [comment]
ok has been given

bors r=rafiss

**Pull request merged**
